app/src/NativeBase/Containers/ProfileContainer.tsx

```diff
@@ -13,20 +44,72 @@ import React from 'react'
 const ProfileContainer = () => {
   return (
     <VStack
-      paddingTop="4"
-      paddingBottom="2"
-      alignItems="center"
-      space="4"
-      backgroundColor="secondaryGrey.60"
+      padding={2}
+      backgroundColor="bg.100"
       _dark={{ backgroundColor: 'bgDarkMode.100' }}
     >
-      <Center
-        backgroundColor="bg.100"
-        _dark={{ backgroundColor: 'bgDarkMode.100' }}
-        width="100%"
-      >
-        <Heading size="sm">My Profile</Heading>
+      <Center>
+        <Heading fontFamily="title" size="lg">
+          My Profile
+        </Heading>
+        <Box
+          marginTop={6}
+          height="93px"
+          width="90px"
+          borderColor="darkGrey.100"
+          borderWidth="4px"
+          borderRadius="full"
+        />
+        <HStack>
+          <Heading fontSize="md" fontWeight="700">
+            Full Name
+          </Heading>
+          <Text fontSize="xs" paddingLeft="-2">
+            (She)
+          </Text>
+        </HStack>
+        <Text fontSize="xs" marginTop="-4">
+          P-R-O-N-O-U-N-C-I-A-T-I-O-N
+        </Text>
+        <Text fontSize="xs" paddingTop={-2}>
+          Role
+        </Text>
+        <Text fontSize="xs" paddingTop={-2}>
+          Status
+        </Text>
+        <Heading fontSize="md" fontWeight="700" paddingTop={-2}>
+          My volunteering
+        </Heading>
+        <Box
```

Comment:
Would this work as a `Card` and then it would do things like shadow automatically for you?

Reply:
The card has a corner radius, a different background colour and a border instead of shadow. Could use that but with so many changes it works fine with Box too :)

Reply:
Got it -- makes sense, thanks

Reply:
Another thought - is this box style being used elsewhere in any of the new designs?  And if so, could you make it into a reusable component?